policy/modules/services/spamassassin.if

```diff
@@ -196,6 +196,7 @@ interface(`spamassassin_exec_client',`
 
 	corecmd_search_bin($1)
 	can_exec($1, spamc_exec_t)
+	read_lnk_files_pattern($1, spamc_exec_t, spamc_exec_t)
```

Comment:
There should be no `spamc_exec_t` symlinks. The labeling should be fixed instead.

Reply:
OK. I will split that out into its own commit, since it is primarily for `rspamd`.